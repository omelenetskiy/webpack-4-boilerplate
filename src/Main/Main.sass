.main
  background-color: $main-background-color
  color: $main-color
  display: flex
  font-size: 24px
  font-weight: bold
  justify-content: space-between
  margin-bottom: 20px
  padding: 20px

.item
  background-color: blue
