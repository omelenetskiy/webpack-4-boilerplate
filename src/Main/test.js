const a = 10;
const b = 20;
let d = 1;
