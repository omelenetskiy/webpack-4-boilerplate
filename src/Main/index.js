import React, { Component } from 'react';
import './Main.sass';
import logo from '../static/img/react/logo.png';

class Main extends Component {
	state = {};
	render() {
		const { title } = this.props;
		return (
			<div>
				<h2>{title}</h2>
				<img src={logo} alt="" />
			</div>
		);
	}
}
export default Main;
