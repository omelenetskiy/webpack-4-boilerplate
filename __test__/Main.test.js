import React from 'react';
import Main from '../src/Main';
import { render, shallow } from 'enzyme';

describe('<Main />', () => {
  it('should render a Main', () => {
    const wrapper = shallow(<Main />);
    expect(wrapper).toMatchSnapshot();
  });
});
