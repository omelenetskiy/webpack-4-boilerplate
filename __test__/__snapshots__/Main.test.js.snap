// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render a Main 1`] = `
<div>
  <h2 />
  <img
    alt=""
    src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mNk+P+/HgAFhAJ/wlseKgAAAABJRU5ErkJggg=="
  />
</div>
`;
